src/popup/components/NotificationItem.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     class="notification-item"
-    @click.stop="handleClick"
+    @click.stop="(event) => handleClick(event)"
```

Comment:
Can't we just write?
```suggestion
    @click.stop="handleClick"
```